src/main/java/apoc/export/csv/CsvFormat.java

```diff
@@ -129,8 +129,8 @@ public void writeAll(SubGraph graph, Reporter reporter, ExportConfig config, CSV
         out.writeNext(header.toArray(new String[header.size()]), applyQuotesToAll);
         int cols = header.size();
 
-        writeNodes(graph, out, reporter, nodePropTypes, cols, config.getBatchSize(), config.getDelim());
-        writeRels(graph, out, reporter, relPropTypes, cols, nodeHeader.size(), config.getBatchSize(), config.getDelim());
+        writeNodes(graph, out, reporter, nodeHeader.subList(2, nodeHeader.size()), cols, config.getBatchSize(), config.getDelim());
```

Comment:
this looks very implict, can we make it explicit, what it means? the offset of 2 or 3